strict-version-matcher-plugin/src/main/java/com/google/android/gms/dependencies/DependencyInspector.java

```diff
@@ -116,7 +116,10 @@ private void registerDependencies(@Nonnull ResolvableDependencies resolvableDepe
           // TODO(paulrashidi): Figure out if a third level dependency shows depFromString.
           // In a project with other project dependencies the dep
           // string will be "project :module1"
-          String depName = depFromString.split(":")[1];
+          // Sometimes depFromString is just "project:" and in that case we want the name
+          // to just be an empty string.
```

Comment:
In the comment you say that you want the name to be an empty string, but in the code, if the value **is actually empty**, you set it to `"module"`